lib/rspec/core/bisect/runner.rb

```diff
@@ -67,17 +74,22 @@ def run_locations(*capture_args)
           end
         end
 
+        # Consider just using Process.spawn since Open3 uses that.
```

Comment:
Yes, was my first attempt but a lot broke, so I revised strategy to look for how I could test it... In the end, decided to make a pr to let you see what I am thinking and trying, and if you have any ideas which is right place to test.... 

Thanks for looking

 Because I'm testing bisect, I can't run the results of a bisect on any failures :)

B mobile phone

> On Jun 4, 2016, at 12:37 PM, Myron Marston <notifications@github.com> wrote:
> 
> In lib/rspec/core/bisect/runner.rb:
> 
> > @@ -67,17 +74,22 @@ def run_locations(*capture_args)
> >            end
> >          end
> >  
> > +        # Consider just using Process.spawn since Open3 uses that.
> Did you try using Process.spawn? Is there a reason you can't use that instead of mutating ENV['SPEC_OPTS']?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>